### DISCUSSION TITLE
Loop sounds

### DISCUSSION THREAD

**Discussion by NatulilakkReal** [SkriptLang/Skript#5785] [General]
I want to make a skript where youre in a world and a sound that last maby 30 sec and it repeats after the sound is done like loops 1 or more sounds in a world

**Comment by AyhamAl-Ali** [comment]
This isn't the correct place to ask for Skript coding help. Please use [skUnity discord server](https://discord.gg/skript) for that.

**Reply by NatulilakkReal** [reply]
oh thanks
